src/coreclr/src/jit/importer.cpp

```diff
@@ -15146,14 +15146,18 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                     {
                         const ssize_t allocSize = op2->AsIntCon()->IconValue();
 
+                        bool bbInALoop =
+                            (compIsForInlining() && ((impInlineInfo->iciBlock->bbFlags & BBF_BACKWARD_JUMP) != 0)) ||
```

Comment:
Should we use a helper method to check this property here, as this could be needed in more places in the future?

Reply:
Good suggestion. It is already used elsewhere.